hoprnet/hoprnet
# frontend security

How to test:

1. Launch `hoprd` as usual, provide `--apiToken <your_token>`
2. Open Admin Panel.
3. Token input field should appear in the bottom
4. Enter wrong token. 
5. It should not let you in.
6. Enter correct token.
7. Admin panel should start working.
